rabbitmq/datadog_checks/rabbitmq/rabbitmq.py

```diff
@@ -534,7 +534,9 @@ def _get_metrics(self, data, object_type, custom_tags):
             for path in keys[:-1]:
                 root = root.get(path, {})
```

Comment:
@ian28223, Doesn't this fail when trying to create message_details from queue_totals?

Reply:
Followup PR https://github.com/DataDog/integrations-core/pull/4805

Reply:
I think this will not fail but value will be `None` and therefore will not get any `queue_totals/* metrics`. Couldn't find any documentation showing what data looked like in queue_totals for that version.

Reply:
Did some further testing and found that `queue_totals` is a blank list `[]` only at the beginning when no queues have been created yet. Once a queue has been created then it becomes a dict even if all queues have been purged and/or deleted. It becomes a blank list again if and only if all queues have been deleted and rabbitmq-server is restarted.

Reply:
This test fails currently on master: https://github.com/DataDog/integrations-core/pull/4805/files#diff-14e06028380f56a0b9ecc2df6cda6114R97